shared/chat/conversation/input-area/mention-handler-hoc/index.native.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import * as React from 'react'
+import {type Props} from '../normal'
+import {type PropsFromContainer} from '.'
+
+type MentionHocState = {
+  pickSelectedCounter: number,
+  mentionFilter: string,
+  channelMentionFilter: string,
+  mentionPopupOpen: boolean,
+  channelMentionPopupOpen: boolean,
+  _selection: {selectionStart: number, selectionEnd: number},
+}
+
+const mentionHoc = (InputComponent: React.ComponentType<Props>) => {
+  class MentionHoc extends React.Component<PropsFromContainer, MentionHocState> {
```

Comment:
🍳 The channel mention HUD doesn't seem to work on mobile at all

Reply:
fixed in 10: https://github.com/keybase/client/pull/10755/files#diff-17534f3bd85a2c3189f96e26e60b5bc1R51